### PR TITLE
Challenge/2/hooks

### PR DESCRIPTION
**Pull request by Gonzz-96**

### CODE REVIEW

**Comment by Gonzz-96** [comment]
Hello, reviewer! In my Google form submission I wrote that my site didn't work in production, but it worked locally. I found the reason to be the API key, which was missing in my Netlify configuration. 🎉

File: src/components/Suggestions/styled.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import styled from 'styled-components';
+
+const UList = styled.ul`
+  flex: 1;
+  padding: 0px;
+  list-style-type: none;
+`;
```
Comment (PacoMojica) [inline src/components/Suggestions/styled.js]: This one works just fine, you can use more expressive names, maybe something like `VideoList` or 'SuggestionContainer`, choosing a name that tells the reader about the purpose of the component improves the readability

File: src/components/VideoSuggestion/styled.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import styled from 'styled-components';
+
+const Container = styled.div`
+  width: 100%;
+  display: flex;
+  padding-bottom: 5px;
+`;
```
Comment (PacoMojica) [inline src/components/VideoSuggestion/styled.js]: I like how this one works, you can improve the UX a lot by adding one line: 
```suggestion
const Container = styled.div`
  width: 100%;
  display: flex;
  padding-bottom: 5px;
  cursor: pointer;
`;
```
With `cursor: pointer;` you let the user know that if the card is clicked something will happen, just a little detail

File: src/components/_tests/Header.spec.jsx [hunk 3]
```diff
@@ -23,4 +23,32 @@ describe('Tests on VideoCard component', () => {
...
+  });
+
+  it('Check onChange working correctly', () => {
+    const value = 'Hello, world';
+
+    const onSubmit = (text) => {
+      expect(text).toBe(value);
+    };
+
+    const { container } = render(<Header onSubmit={onSubmit} />);
```
Comment (PacoMojica) [inline src/components/_tests/Header.spec.jsx:47]: Great, testing new functionally helps in the long run

File: src/hooks/useVideosSearch.js [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+      }
+    };
+
+    if (process.env.NODE_ENV === 'development' || process.env.NODE_ENV === 'test') {
+      setVideos(items);
+    } else {
+      fetchVideos();
+    }
+  }, [keyword]);
+
```
Comment (PacoMojica) [inline src/hooks/useVideosSearch.js:30]: Awesome custom hook to fetch data, you also added some tests for the hook 👍

File: src/pages/Home/Home.page.jsx [hunk 5]
```diff
@@ -1,29 +1,54 @@
...
+                }}
+                description={description}
+                thumbnail={thumbnails.high.url}
+              />
+            );
+          })}
+        </GridContainer>
+      </React.Fragment>
+    );
+  } else {
```
Comment (PacoMojica) [inline src/pages/Home/Home.page.jsx:42]: Here you have an `if/else` statement to check if the user is in the home page or in a specific video, with `Routing` you can use a route switch to handle this logic, but it is something that will be part of the next topics

**Review by PacoMojica — COMMENTED** [review]
Your challenge looks good, I made a couple of suggestions but just minor things. Also I'm glad that you were able to find the problem with the deployment, I've been in similar situations before 😆
Great work!

File: src/components/VideoSuggestion/styled.js (hunk 2 above)
Comment (Gonzz-96) [reply src/components/VideoSuggestion/styled.js]: Awesome! I also added this property to my `VideoCard`. Great suggestion!

**Comment by Gonzz-96** [comment]
@PacoMojica thanks for your comments! I added your suggestions to my code. 😄

**Pull request merged**